src/vue/components/infl-attribute.vue

```diff
@@ -69,7 +71,18 @@ export default {
         decorated = `(${decorated})`
       }
       if (this.decorators.includes('brackets')) {
-        decorated = `[${decorated}]`
+        if (!this.decorators.includes('appendcomma')) {
```

Comment:
I don't understand the name "appendcomma" for this decorator. it's joining values by spaces isn't it?

Reply:
Changed to `appendspace` (it was joined by comma in the first version, then I changed to space and missed to change the title